lib/flight_job/cli.rb

```diff
@@ -75,6 +75,10 @@ def self.create_command(name, args_str = '')
     global_slop.bool '--ascii', 'Display a simplified version of the output, when supported'
     global_slop.bool '--json', 'Display a JSON version of the output, when supported'
 
+    # NOTE: NEXT MAJOR CLI RELEASE
+    # Please review the Outputs and <Model>#serializable_hash methods on the next major release
+    # These will need simplifying
+
```

Comment:
Can you explain this please?

Reply:
Just a reminder that the `cli.rb` isn't the only file that makes up the public interface for `semvar` purposes.

If and when we look at making a major version bump, we have the opportunity to:
* Reorder the "human" and "machine" outputs to be consistent, and
* Modify the `keys` which appear in the `--json` output.

---

This is not to say we _should_ make those changes on a major version bump, but we _could_ do so.

Reply:
Instead of comments like this littered about the source code, I'd suggest a document listing what changes we consider acceptable (or unacceptable) for each of the semvar major, minor, patch releases.  I'll add such a document as part of the README consistency work.